src/components/forms/Buses.js

```diff
@@ -35,7 +35,7 @@ const Buses = ({
     } catch (error) {
       const validationErrors = error.response.data.data;
       if (validationErrors) {
-        for (let field in validationErrors) {
+        for (const field in validationErrors) {
```

Comment:
If you can manage to fix this issue since we are getting structured response from the backend it would be great

Reply:
Will look into it